### ISSUE TITLE
[web] Deleted project not removed from recent projects list

### ISSUE THREAD

**Issue by rossmcdonald** [stoplightio/studio#66]
**Describe the bug**

When removing project from settings menu, project is not removed from recent projects listing on Studio landing screen.

**To Reproduce**

1. Within an open project in Studio web, go to Settings > Project Settings > Delete Project

**Expected behavior**

Project would be removed from project listing.

**Additional Context**

Please also verify that in studio desktop removing a project works as expected (with and without remove files option).

**Comment by chris-miaskowski** [comment]
@rossmcdonald , @chohmann 

I have dug into the code and discovered that in latest version deleting projects were disabled.

https://github.com/stoplightio/studio-internal/blob/329c71ca1c243035b40289de1e99644cd9b85357/src/components/SettingsDialog/ProjectPanel.tsx#L69

```
{process.env.RUN_CONTEXT === 'desktop' && (
        <FormGroup
          labelFor="delete-project-button"
          intent={Intent.DANGER}
          helperText="Removes the current project and optionally remove the associated file directory."
        >
          <Button
            id="delete-project-button"
            icon="trash"
            intent={Intent.DANGER}
            text="Delete Project"
            onClick={onDeleteProjectBtnClick}
          />
          <RemoveProjectDialog isOpen={isRemoveProjectDialogOpen} onClose={onProjectDeletionModalClose} />
        </FormGroup>
      )}
```

Is that related to the recent project settings changes? It this `bug` still relevant?

CC: @casserni @StefanDywersant @marbemac

**Comment by chohmann** [comment]
@rossmcdonald @chris-miaskowski In production right now, the Settings cog displayed in the bar is coming from Studio (so not the new project settings that I implemented). I can't speak for if those settings should actually delete the project from the Studio cache, removing it from your recent projects.

Th new project settings have been deployed to integration. I just confirmed again, that when using the Delete project from the new project settings, the project is successfully getting removed from the studio cache, removing it from your recent projects. 

As part of the new project settings, I moved the Studio settings to the metabar (bottom bar), and hid the Delete Project button in these setting is web context, since the new project settings also has a delete project option.

![image](https://user-images.githubusercontent.com/33431856/63956276-c2bc7780-ca4b-11e9-9fbc-c920274090a8.png)

![image](https://user-images.githubusercontent.com/33431856/63956404-f9928d80-ca4b-11e9-9fb1-274f66065819.png)

![image](https://user-images.githubusercontent.com/33431856/63956548-3fe7ec80-ca4c-11e9-86a5-8e267242437d.png)


This could still be an issue when using desktop mode, since the original studio settings delete project is still used in desktop context, and it might not be deleting the project from studio's cache. So I would test in desktop mode, and if it doesn't happen there, then this can be closed.
